vendor/brave-ios/tests/ads_wrapper_test.mm

```diff
@@ -84,4 +86,11 @@ - (void)testServeSampleAd
   [self waitForExpectations:@[expect] timeout: 4.0];
 }
 
+#pragma mark - BATBraveRewardsDelegate
+
+- (void)faviconURLFromPageURL:(NSURL *)pageURL completion:(void (^)(NSURL * _Nullable))completion
+{
+  completion(nil);
```

Comment:
How come it only returns nil? Or is it a TODO or?

Reply:
Yeah test case doesnt need to fetch favicons :)